linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_encrypt_sent_records/rule.yml

```diff
@@ -29,6 +29,7 @@ identifiers:
 
 references:
     disa: "1851"
+    nist: AU-9(3)
```

Comment:
Ack

Ref: https://nvd.nist.gov/800-53/Rev4/control/AU-9#enhancement-3

"The information system implements cryptographic mechanisms to protect the integrity of audit information and audit tools."